contrib/systemd/system/podman-restart.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Podman Start All Containers With Restart Policy Set To Always
+Requires=podman.socket
+After=podman.socket
+Documentation=man:podman-start(1)
+StartLimitIntervalSec=0
+
+[Service]
+Type=fork
```

Comment:
`s/fork/forking`  but I actually think this should be `type=simple`.

Reply:
Why?

Reply:
Reread the man page.  Indeed oneshot seems better for error reporting at startup.

Reply:
TIL. Thanks. I will update this soon.